src/sasctl/utils/functionInspector.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
```

Comment:
Per PEP8 module names should be lower case and include underscores if necessary.
[https://peps.python.org/pep-0008/#package-and-module-names](https://peps.python.org/pep-0008/#package-and-module-names)

Reply:
Definitely outdated and should not have been included in the git commits. Removing it and resolving any other comments based on functionInspector.py.